js/renovation/ui/scroll_view/scroll_view.tsx

```diff
@@ -212,6 +212,16 @@ export class ScrollView extends JSXComponent<ScrollViewPropsType>() {
     // TODO: this.clientHeight() should be containerRef.current.clientHeight
   }
 
+  @Method()
+  startLoading(): void {
```

Comment:
For
"if(!scrollView || !scrollView.startLoading) {"
https://github.com/DevExpress/DevExtreme/blob/1b45e331d1eed5c793355331bb2b5c05fd7b7ec9/js/ui/tile_view.js#L141
and 
"this._scrollView?.startLoading();"
https://github.com/DevExpress/DevExtreme/blob/689acd2c5a46ff21f2d2767549f00f31554a2e7a/js/ui/list/ui.list.base.js#L488